src/biokbase/narrative/tests/test_jobmanager.py

```diff
@@ -776,14 +776,11 @@ def mock_check_job(params):
         reg_child_jobs = [
             self.jm.get_job(job_id) for job_id in batch_job._acc_state["child_jobs"]
         ]
-
         self.assertCountEqual(batch_job.children, reg_child_jobs)
-        self.assertCountEqual(batch_job._acc_state["child_jobs"], new_child_ids)
```

Comment:
I think this can go back in, but ofc like `self.assertCountEqual(batch_job.child_jobs, new_child_ids)`